appveyor.yml

```diff
@@ -122,7 +122,7 @@ test_script:
      --n-retries 2
      test\command\suite
   - ruby test\command_line\run-test.rb
-     --groonga-install-prefix=%FULL_GROONGA_INSTALL_FOLDER%
+     --groonga-install-prefix=%APPVEYOR_BUILD_FOLDER%\\\u7E67\uFF64\u7E5D\uFF73\u7E67\uFF79\u7E5D\u533B\u30FB\u7E5D\uFF6B\\%GROONGA_INSTALL_FOLDER%
```

Comment:
`\\\u7E67\uFF64\u7E5D\uFF73\u7E67\uFF79\u7E5D\u533B\u30FB\u7E5D\uFF6B\\` is difficalt to understand.
Therefore, in my opinion, we should set the above value to a variable with a comment as below example.

```
# On Appveyor, インストール directory is recognized unicode escape sequence.
set GROONGA_INSTALL_SUB_FOLDER_ESCAPE=\\\u7E67\uFF64\u7E5D\uFF73\u7E67\uFF79\u7E5D\u533B\u30FB\u7E5D\uFF6B\\
```

Reply:
@komainu8 it is reasonable to fix it.